### ISSUE TITLE
inaccurate warning message

### ISSUE THREAD

**Issue by tabVersion** [microsoft/nni#2847]
**Environment**:
- NNI version: v1.7
- NNI mode (local|remote|pai): local & pai
- Client OS: Ubuntu 16.04
- Server OS (for remote mode only):
- Python version: 3.7
- PyTorch/TensorFlow version: pyTorch 1.5
- Is conda/virtualenv/venv used?: conda
- Is running in Docker?: no

**Log message**:
The following warning message was shown on console. 
 ```bash
$ nnictl create --config BOHB_config.yml --port 12340
INFO:  expand searchSpacePath: search_space.json to /data/data6/v-bzhang/search_space/nni_search_space/examples/trials/sklearn/classification/search_space.json
INFO:  expand codeDir: . to /data/data6/v-bzhang/search_space/nni_search_space/examples/trials/sklearn/classification/.
 
[08/31/2020, 02:53:32 PM] WARNING (nni.protocol) IPC pipeline not exists, maybe you are importing tuner/assessor from trial code?
/home/quzha/v-bzhang/anaconda/envs/nb/lib/python3.6/site-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing as tm
```

**What issue meet, what's expected?**:

This problem occurred when I want to use BOHB on the given Sklearn example. I got the warning message but the experiment went well without informing user. 

**How to reproduce it?**: 

The experiment code is [here](https://github.com/microsoft/nni/blob/master/examples/trials/sklearn/classification/main.py)
And [here](https://github.com/microsoft/nni/pull/2844/files) is the config file. 

**Additional information**:

The incorrect warning comes from https://github.com/microsoft/nni/blob/master/src/sdk/pynni/nni/protocol.py#L33.

**Comment by tabVersion** [comment]
@liuzhe-lz

**Comment by AkshayRoyal** [comment]
I too have the same issue.
**error log:**
```/home/fu-workstation/anaconda3/envs/tf23/lib/python3.8/site-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing as tm
/home/fu-workstation/anaconda3/envs/tf23/lib/python3.8/site-packages/statsmodels/compat/pandas.py:23: FutureWarning: The Panel class is removed from pandas. Accessing it from the top-level namespace will also be removed in the next version
  data_klasses = (pandas.Series, pandas.DataFrame, pandas.Panel)
```
This is just a warning but nni still works post this warning.

**Steps to reproduce**
run mnist-advisor example which uses BOHB

**Comment by liuzhe-lz** [comment]
Fixed in v1.9.
